packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -2776,19 +2776,28 @@ export const isGETDashboardsAgentStatusParams = ajv.compile<GETDashboardsAgentSt
 	GETLivechatAnalyticsDashboardsAgentStatusParamsSchema,
 );
 
-type PUTLivechatPriority = {
-	name: string;
-};
+type PUTLivechatPriority =
+	| {
+			name?: string;
+			reset: boolean;
+	  }
+	| {
+			name: string;
+			reset?: boolean;
+	  };
```

Comment:
🤔 wouldn't be better
```
{ name: string } | { reset: true }

Reply:
not a good idea, it breaks the type checking on the endpoint, went with {name?: string; reset?: boolean;}